app-admin/graylog2/files/initd-r2

```diff
@@ -0,0 +1,31 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+command="/usr/bin/java"
+command_args="${JAVA_OPTS} -jar ${GRAYLOG_INSTALL_DIR}/graylog.jar server -f ${GRAYLOG_CONFIG_FILE} --no-pid-file ${GRAYLOG_OPTIONS}"
+command_background="true"
+command_user="${GRAYLOG_USER}:${GRAYLOG_GROUP}"
+pidfile="/run/${RC_SVCNAME}.pid"
+
+output_log="${GRAYLOG_LOG_DIR}/graylog.stdout.log"
+error_log="${GRAYLOG_LOG_DIR}/graylog.stderr.log"
+directory="${GRAYLOG_INSTALL_DIR}"
+required_files="${GRAYLOG_CONFIG_FILE}"
+
+depend() {
+	need net
+}
+
+start_pre() {
+	local d
+	for d in \
+		"${GRAYLOG_DATA_DIR}" \
```

Comment:
Does this package support systemd? Wondering if we shouldn't use tmpfile service for that. Of course, if it is common to have multiple instances/different DATA_DIRs, it would be handy to keep this.

Reply:
Not yet as it seems none of the maintainers use it and users require it.